CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # Version changelog
 
+## 0.213.0
+
+CLI:
+ * Ignore environment variables for `auth profiles` ([#1189](https://github.com/databricks/cli/pull/1189)).
+ * Update LICENSE ([#1013](https://github.com/databricks/cli/pull/1013)).
```

Comment:
To be more clear, this should say "Update LICENSE file to match DB license language"